_data/init-cli/docker_init.yaml

```diff
@@ -20,13 +20,15 @@ long: |-
   After running `docker init`, you can choose one of the following templates:
     
     * Go: Suitable for a Go server application.
+    * Python: Suitable for a Python server application.
+    * Node: Suitable for a Node server application.
     * Other: General purpose starting point for containerizing your application.
   
   After `docker init` has completed, you must modify the created files and tailor them to your project. Visit the following topics to learn more about the files:
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion:** ‏Unrelated to this PR, so feel free to ignore.
```suggestion
  After `docker init` has completed, you may need to modify the created files and tailor them to your project. Visit the following topics to learn more about the files:
```